app/dashboard/templates/process_bounty.html

```diff
@@ -115,4 +156,14 @@ <h3>{% trans "Basic Payout" %}</h3>
   <script src="{% static "v2/js/tokens.js" %}"></script>
   <script src="{% static "v2/js/pages/shared_bounty_mutation_estimate_gas.js" %}"></script>
   <script src="{% static "v2/js/pages/process_bounty.js" %}"></script>
+  <script type="text/javascript">
```

Comment:
Can we move this to the `bulk_payout.js` ?

Reply:
`bulk_payout.js` is not included in process_bounty.html. Maybe move it to a completely separate file?